app/views/schools/on_boarding/candidate_requirements_selections/_form.html.erb

```diff
@@ -12,7 +12,7 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l">
-      Select candidate requirements
+      What requirements do you have?
```

Comment:
We should also remove this line from the candidate requirements page:

![Screenshot 2021-12-12 at 15 24 49](https://user-images.githubusercontent.com/85290597/145718531-78fb5d5c-875f-4993-862a-2f4d03deeac7.png)